app/assets/stylesheets/organizations.scss

```
@@ -0,0 +1,997 @@
+    .organizations-btn-secondary {
+      text-align: center;
+      width: 100%;
+    }
+  }
+}
+.org-dashboard-wrapper {
+  margin: 0 auto;
+  max-width: 1200px;
+  padding: 30px 20px 0;
+  color: #2c3e50;
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)